inc/thorvg.h

```diff
@@ -1005,6 +1005,7 @@ class TVG_EXPORT Picture final : public Paint
      *
      * @param[in] data A pointer to a memory location where the content of the picture file is stored.
      * @param[in] size The size in bytes of the memory occupied by the @p data.
+     * @param[in] mimetype Mimetype of data. If null or unknown loaders will be tried one by one.
```

Comment:
We can provide better explicit description.

Mimetype of data => Mimetype of data such as "jpg", "svg", "png", etc.

If null => If empty string or @c nullptr